includes/helper-functions.php

```diff
@@ -0,0 +1,305 @@
+<?php
+/**
+ * Helper functions font settings.
+ *
+ * @package     Bsf_Custom_Fonts
+ * @since       x.x.x
+ */
+
+// Exit if accessed directly.
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+/**
+ * Return image media ID on basis of file URL.
+ *
```

Comment:
@params are not added to the function @imnavanath

Reply:
DONE